flink-connector-tidb-cdc/src/main/java/com/ververica/cdc/connectors/tidb/TiKVRichParallelSourceFunction.java

```diff
@@ -198,6 +207,8 @@ protected void readSnapshotEvents() throws Exception {
 
     protected void readChangeEvents() throws Exception {
         LOG.info("read change event from resolvedTs:{}", resolvedTs);
+        // child thread to sink committed rows.
+        new Thread(this::asyncSink).start();
```

Comment:
Use `Executors.newSingleThreadExecutor(threadFactory)`  will be better.

Reply:
good idea, done